gapipy/resources/dossier/activity_dossier.py

```diff
@@ -23,3 +24,42 @@ class ActivityDossier(Resource):
         ('start_location', 'Place'),
         ('end_location', 'Place'),
     ]
+
+    _model_collection_fields = [
+        ('details', DossierDetail),
+    ]
+
+    def __repr__(self):
+        return '<{} {}>'.format(self.__class__.__name__, self.name)
+
+    @property
+    def duration(self):
+        """
+        Simulate the duration dict in `ItineraryComponent` objects,
```

Comment:
Agree! I actually made that mistake myself during development.

I'm not familiar enough to know if that's easy when the data goes that deep. I'll take a peek.